pylinac/ct.py

```diff
@@ -248,8 +250,12 @@ def _setup_rois(self) -> None:
         for name, setting in self.background_roi_settings.items():
             self.background_rois[name] = HUDiskROI(self.image, setting['angle_corrected'], setting['radius_pixels'], setting['distance_pixels'],
                                         self.phan_center)
-        background_median = np.mean([roi.pixel_value for roi in self.background_rois.values()])
-        background_std = np.std([roi.pixel_value for roi in self.background_rois.values()])
+        if self.background_rois:
+            background_median = np.mean([roi.pixel_value for roi in self.background_rois.values()])
```

Comment:
change to mean per your own comment and all good

Reply:
Done. Thanks.